mlrun/projects/project.py

```diff
@@ -1054,23 +991,43 @@ def workflows(self) -> list:
         """This is a property of the spec, look there for documentation
         leaving here for backwards compatibility with users code that used MlrunProjectLegacy"""
         warnings.warn(
-            "This is a property of the spec, use project.spec.workflows instead"
+            "This is a property of the spec, use project.spec.workflows instead. "
             "This will be deprecated in 0.7.0, and will be removed in 0.9.0",
             # TODO: In 0.7.0 do changes in examples & demos In 0.9.0 remove
-            PendingDeprecationWarning,
+            FutureWarning,
         )
         return self.spec.workflows
 
     @workflows.setter
     def workflows(self, workflows):
         warnings.warn(
-            "This is a property of the spec, use project.spec.workflows instead"
+            "This is a property of the spec, use project.spec.workflows instead. "
             "This will be deprecated in 0.7.0, and will be removed in 0.9.0",
             # TODO: In 0.7.0 do changes in examples & demos In 0.9.0 remove
-            PendingDeprecationWarning,
+            FutureWarning,
         )
         self.spec.workflows = workflows
 
+    @property
+    def default_image(self) -> str:
+        return self.spec.default_image
+
+    def set_default_image(self, default_image: str):
```

Comment:
why not make it a property setter?

Reply:
As discussed - since this has some internal logic (replacing existing value etc.) it's better to make it a function, rather than use a setter which is assumed to be "pass-through".